test/test_utils/role_utils.ts

```diff
@@ -100,4 +101,29 @@ export async function requestRole({
     canonizeSig(agreement),
     canonizeSig(proof)
   )).wait(); // testing on Volta needs at least 2 confirmation
-}
\ No newline at end of file
+}
+
+export async function revokeRole({
+  revocationRegistry,
+  revoker,
+  subject,
+  subjectRole,
+  revokerRole,
+  } : {
+    revocationRegistry : RevocationRegistry,
+    revoker : Signer,
+    subject : Signer,
+    subjectRole : string,
+    revokerRole : string
+  }) : Promise<void> {
+
+    const revokerAddr = await revoker.getAddress();
+    const subjectAddr = await subject.getAddress();
+
+    await (await revocationRegistry.revokeClaim(
+      utils.namehash(subjectRole+subjectAddr),
```

Comment:
Why `namehash` instead of `keccak256` here?

Reply:
In my opinion, might as well use `DID` instead of `Addr` here as we are legitimately considering DIDs

Reply:
1. `namehash` was being used in `registerRole` and was easy to use. I would change it `keccak256` implementations.
2. I thought of using `DIDs` on the first, looking at the libraries and other contract (we are using `Addr`) made me think of using `Addr`.